lib/nanoc/base/views/post_compile_item_view.rb

```diff
@@ -0,0 +1,11 @@
+module Nanoc
+  class PostCompileItemView < Nanoc::ItemView
```

Comment:
:+1: for having `PostCompileItemView`. I got a little lazy and didn’t create a `PreprocessItemView` (and friends, like the collection view), but that’ll change Soon™. (See #571.)